src/main/webapp/static/js/services/scheduling_service.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * 
+ */
+ 
+ 'use strict';
+ 
+ angular
+ 	.module('myApp')
+ 	.factory('SchedulingService', SchedulingService)
+	
+SchedulingService.$inject = ['$http', '$log']
+
+function SchedulingService($http, $log) {
+	var REST_SERVICE_URI = 'http://localhost:8080/ScheduleNow/scheduling/jobs/';
```

Comment:
don't ever hardcode the full protocol//host:port/context,    Only ever use the relative path  "scheduling/jobs"